Pear/Models/User.swift

```diff
@@ -16,15 +16,26 @@ struct User: Codable, Equatable {
     let hometown: String
     let profilePictureURL: String
     let major: String
-    let graduationYear: String?
+    let graduationYear: String
     let pronouns: String
     let interests: [String]
     let groups: [String]
     let facebook: String?
     let instagram: String?
 
     static func == (lhs: User, rhs: User) -> Bool {
-        lhs.netID == rhs.netID
```

Comment:
Could you explain this change? Under what circumstance would you compare these two objects? I ask because comparing equality can get confusing -- i.e. what does "equal" mean? My initial reaction was that the object represents the same user (same net id, like you had before), but that clearly is not the case.

Reply:
@danielVebman Yea, so it's because we have some weird implementation that only refreshes our home screen if our `User` object has changed. So even if the user itself is the same (same net ID), we want to refresh as long as any of the fields have changed because we want the changes to be reflected in our side panel, edit views, etc.

Reply:
Is that a library that we use or did we write that implementation ourselves. If we did it ourselves, then maybe we should rewrite it at some point so that it isn't "weird."

Reply:
just a lot of legacy code between ios and backend